web/packages/hovercards/src/style.scss

```
@@ -179,11 +222,135 @@ $color-blue: #1d4fc4;
 
 	.gravatar-hovercard__inner {
 		align-items: center;
-		justify-content: start;
+		justify-content: center;
 		gap: 34px;
 	}
 
 	.gravatar-hovercard__error-message {
 		color: $color-gray;
 	}
 }
+
+.gravatar-hovercard__drawer {
+	position: absolute;
+	top: 0;
+	left: 0;
+	bottom: 0;
+	right: 0;
+	visibility: hidden;
+	overflow: hidden;
+	z-index: 1;
+
+	.gravatar-hovercard__drawer-backdrop {
+		position: absolute;
+		top: 0;
+		left: 0;
+		bottom: 0;
+		right: 0;
+		background-color: rgba($color-black, 0.4);
+		opacity: 0;
+		transition: opacity 0.3s ease-in-out;
+	}
+
+	.gravatar-hovercard__drawer-card {
+		position: absolute;
+		left: 0;
+		bottom: 0;
+		width: 100%;
+		max-height: 100%;
+		display: flex;
+		flex-direction: column;
+		background-color: $color-white;
+		border-top-left-radius: 4px;
+		border-top-right-radius: 4px;
+		padding: 20px 0;
+		box-sizing: border-box;
+		transform: translateY(100%);
+		transition: transform 0.3s ease-in-out;
+	}
+
+	.gravatar-hovercard__drawer-header {
+		display: flex;
+		align-items: center;
+		justify-content: space-between;
+		padding: 0 20px;
+		margin-bottom: 16px;
+	}
+
+	.gravatar-hovercard__drawer-title {
+		font-family: $font-sf-pro;
+		font-size: 18px;
+		font-weight: 700;
+		line-height: 27px;
+		margin: 0;
+	}
+
+	.gravatar-hovercard__drawer-close {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		border: none;
+		background: none;
+		padding: 0;
+		cursor: pointer;
+	}
+
+	.gravatar-hovercard__drawer-items {
+		display: flex;
+		flex-direction: column;
+		list-style: none;
+		margin: 0;
+		padding: 0 20px;
+		gap: 12px;
+		overflow: auto;
+	}
+
+	.gravatar-hovercard__drawer-item {
+		display: flex;
+		gap: 8px;
+	}
+
+	.gravatar-hovercard__drawer-item-info {
+		display: flex;
+		flex-direction: column;
+	}
+
+	.gravatar-hovercard__drawer-item-label {
+		font-weight: 600;
+		line-height: 24px;
+		text-transform: capitalize;
+	}
+
+	.gravatar-hovercard__drawer-item-link {
+		color: $color-blue;
+		text-decoration: none;
+
+		&:hover {
+			text-decoration: underline;
+		}
+	}
+
+	&--open {
+		visibility: visible;
+
+		.gravatar-hovercard__drawer-backdrop {
+			opacity: 1;
+		}
+
+		.gravatar-hovercard__drawer-card {
+			transform: translateY(0);
```

Utilize the GPU to ensure smooth animations:

```suggestion
			transform: translate3d(0, 0, 0);
```